src/PolyType.Roslyn/Helpers/RoslynHelpers.cs

```diff
@@ -220,16 +220,16 @@ type is
     public static bool HasSetsRequiredMembersAttribute(this IMethodSymbol constructor)
     {
         return constructor.MethodKind is MethodKind.Constructor &&
-            constructor.GetAttributes().Any(attr => 
```

Comment:
Ugh. I need to turn on the relevant analyzer for trailing whitespace but keep forgettig to do that.